frontend/src/app/projects/page.tsx

```
@@ -1,16 +1,46 @@
+        position: index + 1,
+        url: `https://luisfaria.dev/projects/${project.slug}`,
+        name: project.title,
+      })),
+    },
+  };
 
   return (
     <MainLayout>
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(itemListLd) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)